e2e/test/iothub/messaging/FaultInjectionPoolAmqpTests.MessageReceiveFaultInjectionPoolAmqpTests.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Azure.Devices.E2ETests
 {
     public partial class FaultInjectionPoolAmqpTests
     {
-        private readonly string MessageReceive_DevicePrefix = $"MessageReceiveFaultInjectionPoolAmqpTests";
+        private readonly string MessageReceive_DevicePrefix = "MessageReceiveFaultInjectionPoolAmqpTests";
```

Comment:
```suggestion
        private readonly string MessageReceive_DevicePrefix = $"{nameof(MessageReceiveFaultInjectionPoolAmqpTests)}_";
```

for consistency with the other tests.

Reply:
That isn't a name of a type. :(